SCP-4389 Update PK to Address (#16)
input-output-hk/marlowe-playground: doc/marlowe/tutorials/javascript-embedding.rst

==============================

Marlowe is written as a Haskell data type, and thus it is
straightforward to describe Marlowe smart contracts using Haskell. But since Marlowe contracts are “just”

> a form of data, we can equally well represent them in other languages that can be serialized as JSON or CBOR.

Reviewer: ```suggestion
a form of data, we can equally well represent them in other languages that support serialization to JSON or CBOR.
```
Me being nitpicky as usual: the language itself does not need to be serialized, just support serializing, right?
